src/components/People.js

```diff
@@ -17,30 +18,30 @@ const People = React.createClass({
         this.state.updated_niceties.forEach(function(e) {
             const split_e = e.split(",");
             let anonymous;
-            if (localStorage.getItem("anonymous-" + split_e[0]) === "undefined" || localStorage.getItem("anonymous-" + split_e[0]) === null) {
-                anonymous = "false";
+            if (store.get("anonymous-" + split_e[0]) === "undefined" || store.get("anonymous-" + split_e[0]) === null) {
+                anonymous = false;
             } else {
-                anonymous = localStorage.getItem("anonymous-" + split_e[0]);
+                anonymous = store.get("anonymous-" + split_e[0]);
             }
```

Comment:
It looks like store2 also allows you to pass an alternative value for it to return if the key is not found:

https://github.com/nbubna/store/blob/553bd9036b1036e86904a2135f72d5497d5c11de/src/store2.js#L127

Which I think would allow this code to be simplified to:

```
const anonymous = store.get("anonymous-" + split_e[0], false);
```